ts/test/data/data_test.ts

```diff
@@ -0,0 +1,60 @@
+import { afterEach, beforeEach, describe } from 'mocha';
+import Sinon from 'sinon';
+import { expect } from 'chai';
+import { Data } from '../../data/data';
+import { ConversationModel } from '../../models/conversation';
+import { channels } from '../../data/channels';
+
+describe('data', () => {
```

Comment:
Very nice! :+1:

Reply:
I've got another PR coming with the rest of this file tested once this is merged 😉